app/controllers/UpscanController.scala

```diff
@@ -163,6 +168,7 @@ class UpscanController @Inject()(repository: UploadJourneyRepository,
     implicit request => {
       S3UploadSuccessForm.upscanUploadSuccessForm.bindFromRequest.fold(
         errors => {
+          PagerDutyHelper.log("filePosted",FILE_VERIFICATION_FAILURE_UPSCAN)
```

Comment:
Not sure on the naming of this - the method suggests it's when the file has been posted, not when it's being verified